src/pip/_internal/cli/index_command.py

```diff
@@ -36,10 +38,13 @@ def _create_truststore_ssl_context() -> Optional["SSLContext"]:
 
     try:
         from pip._vendor import truststore
-    except ImportError as e:
-        raise CommandError(f"The truststore feature is unavailable: {e}")
+    except ImportError:
+        logger.warning("Disabling truststore because platform isn't supported")
+        return None
 
-    return truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+    ctx = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+    ctx.load_verify_locations(certifi.where())
```

Comment:
Just curious, what's the purpose of this addition?

Reply:
This change was done to ensure that this change is backwards compatible, so the now instead of only using certifi the trust store is the union of certifi and the system trust store.

Reply:
Fair enough. I was just confused AFAIU the codebase already falls back to certifi even without this line, but maybe that's not `--use-feature=truststore` did. :+1: